### PR TITLE
Y stream Cluster Logging  upgrade 4.1 to 4.2

### PR DESCRIPTION
**Pull request by mburke5678**
https://mojo.redhat.com/people/anli/blog/2019/09/19/cluster-logging-y-stream-upgrade-41-42

### CODE REVIEW

File: modules/cluster-logging-upgrading-logging.adoc [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+----
+Elasticsearch Operator
+4.2.0-201909201915 provided 
+by Red Hat, Inc
+----
+
+.. Ensure that the Elasticsearch pods are using the 4.2 image:
++
+----
+$ oc get pod -o yaml -n openshift-logging --selector component=elasticsearch |grep 'image:'
```
Comment (anpingli) [inline modules/cluster-logging-upgrading-logging.adoc]: https://bugzilla.redhat.com/show_bug.cgi?id=1756304#c2.  After elasticsearch-operators is upgraded to 4.2. The ose-oauth-proxy is v4.2.0-*,  the elasticearch5 is still v4.1.x-*.  The elasticsearch5 image will be upgraded to 4.2 after clusterlogging is upgraded.

Comment (mburke5678) [reply modules/cluster-logging-upgrading-logging.adoc]: @anpingli Thank you for pointing this out, What if we move the verification steps to the end of the procedure? 
Update ESO using the web console
Update CLO using the web console
Verify the pod upgraded and are active using the CLI

Comment (anpingli) [reply modules/cluster-logging-upgrading-logging.adoc]: That is OK.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: modules/cluster-logging-upgrading-logging.adoc [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+[id="cluster-logging-about_{context}"]
+= Upgrading cluster logging
+
+To upgrade cluster logging from 4.1 to 4.2,  after upgrading the cluster 
+update the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+
+.Prerequisites
+
+* Upgrade cluster from 4.1 to 4.2
```
Comment (bergerhoffer) [inline modules/cluster-logging-upgrading-logging.adoc]: Be consistent in the use of periods at the end or not in this list.

File: modules/cluster-logging-upgrading-logging.adoc [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+* Make sure the clusterlogging status is healthy
++
+** All pods are ready.
```
Comment (bergerhoffer) [inline modules/cluster-logging-upgrading-logging.adoc]: Pods

Throughout PR

File: modules/cluster-logging-upgrading-logging.adoc [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+$ oc get csv
+----
+
+.Procedure
+
+. Upgrade the Elasticsearch Operator:
+
+.. From the web console, click *Operators* -> *Installed Operators*. 
+
+.. Select the `openshift-logging` Project.
```
Comment (bergerhoffer) [inline modules/cluster-logging-upgrading-logging.adoc]: Project -> project

File: modules/cluster-logging-upgrading-logging.adoc [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+.. Click the *Elasticsearch Operator*.
+
+.. Click *Subscription* -> *Channel*. 
+
+.. In the Change Subscription Update Channel window, select *4.2* and click `Save`.
```
Comment (bergerhoffer) [inline modules/cluster-logging-upgrading-logging.adoc]: If Save is a button, should be asterisks for bold too.

Comment (bergerhoffer) [reply modules/cluster-logging-upgrading-logging.adoc]: Also should "Change Subscription Update Channel" be bold if it's a GUI item?

File: modules/cluster-logging-upgrading-logging.adoc [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+.. Wait for a seconds, then click *Operators* -> *Installed Operators*. 
++
+The Elasticsearch Operator is shown a 4.2. For example:
```
Comment (bergerhoffer) [inline modules/cluster-logging-upgrading-logging.adoc]: a -> as?

File: modules/cluster-logging-upgrading-logging.adoc [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
++
+----
+Elasticsearch Operator
+4.2.0-201909201915 provided 
+by Red Hat, Inc
+----
+
+. Upgrade the Cluster Logging Operator:
```
Comment (bergerhoffer) [inline modules/cluster-logging-upgrading-logging.adoc]: Same comments as the above step for this step/substeps.

File: modules/cluster-logging-upgrading-logging.adoc [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+
+{
+  "cluster_name" : "elasticsearch",
+  "status" : "green",
+
+....
+
+----
+
+.. Ensure that the logging collector pods are using to 4.2 images and pods status are ready:
```
Comment (bergerhoffer) [inline modules/cluster-logging-upgrading-logging.adoc]: I don't see anything in the below output for "ready", is this correct to say here?

If so, the wording is a little off (and a "to" typo). Maybe

```
Ensure that the logging collector pods are using 4.2 images and are ready:
```

File: modules/cluster-logging-upgrading-logging.adoc [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-fluentd:v4.2.0-201909201915
+----
+
+.. Ensure that the Kibana pods are using to 4.2 images and pods status are ready:
```
Comment (bergerhoffer) [inline modules/cluster-logging-upgrading-logging.adoc]: Same comments as above step

File: modules/cluster-logging-upgrading-logging.adoc [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+----
+$ oc get pod -n openshift-logging --selector logging-infra=kibana -o yaml |grep 'image:'
+
+image: registry.redhat.io/openshift4/ose-logging-kibana5:v4.2.0-201909210748
+image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.2.0-201909201915
+image: registry.redhat.io/openshift4/ose-logging-kibana5:v4.2.0-201909210748
+image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.2.0-201909201915
+----
+
+.. Ensure that the Curator CronJob is using to 4.2 images and pods status are ready:
```
Comment (bergerhoffer) [inline modules/cluster-logging-upgrading-logging.adoc]: Same comments as above step

File: modules/cluster-logging-upgrading-logging.adoc [hunk 11]
```diff
@@ -0,0 +1,154 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging.adoc
+
+[id="cluster-logging-about_{context}"]
+= Upgrading cluster logging
+
+To upgrade cluster logging from 4.1 to 4.2,  after upgrading the cluster
```
Comment (bergerhoffer) [inline modules/cluster-logging-upgrading-logging.adoc]: Suggest rewording. Maybe:

```
After upgrading the {product-title} cluster, you can upgrade cluster logging from 4.1 to 4.2 by updating the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
```

**Comment by mburke5678** [comment]
@bergerhoffer Thanks for the review. Apparently, it wasn't ready for peer review. Apologies for the typos.
